jhub_apps/service/initialize_startup_apps.py

```diff
@@ -0,0 +1,128 @@
+import asyncio
```

Comment:
How do you feel about moving this to a new directory inside jhub-apps, named something like tasks/commands so that we have a defined way for adding task like services?

Reply:
That's totally fine with me